portfolio/src/main/java/com/google/sps/servlets/DataServlet.java

```diff
@@ -24,9 +36,67 @@
 @WebServlet("/data")
 public class DataServlet extends HttpServlet {
 
+  private List<String> quotes;
+
+  @Override
+  public void init() {
+  }
+
+
+  // Generates a randome quote from the ArrayList Quotes and converts the quote to a JSON object.
   @Override
   public void doGet(HttpServletRequest request, HttpServletResponse response) throws IOException {
-    response.setContentType("text/html;");
-    response.getWriter().println("<h1>Hello Jason</h1>");
+    Query query = new Query("Task").addSort("timestamp", SortDirection.DESCENDING);
+
+    DatastoreService datastore = DatastoreServiceFactory.getDatastoreService();
+    PreparedQuery results = datastore.prepare(query);
+
+    List<Task> tasks = new ArrayList<>();
+    for (Entity entity : results.asIterable()) {
+      String name = (String) entity.getProperty("name");
+      String question = (String) entity.getProperty("question");
+      long timestamp = (long) entity.getProperty("timestamp");
+
+      Task task = new Task(name, question, timestamp);
```

Comment:
Each task here seems to be a question and not really a task. Could you rename all instances (across your files) of `Task`/ `task` to `Question`/`question` so it's clearer what you're storing?